docs/docs/marathon-ui.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Web UI
+---
+
+# Marathon UI
+
+The **Marathon UI** source code repository lives at [https://github.com/mesosphere/marathon-ui](https://github.com/mesosphere/marathon-ui).
+For issues and feature requests related to the web UI, please refer to the [Support]({{ site.baseurl }}/support.html) page.
+
+<img src="{{ site.baseurl }}/img/marathon-ui-0.13.png" width="700" alt="Marathon UI in 0.13">
+
+Marathon exposes its web UI on port `8080`.
+
+## Application Status Reference
+Marathon UI introduces the following concepts to illustrate the possible statuses
+an app can be in at any point in time:
+
+- **Running**
+- **Deploying**
+- **Suspended**
+- **Delayed**
+- **Waiting**
+
+These statuses are displayed in the UI's application list view to provide an
+at-a-glance overview of the global state to the user.
+
+Following is an explanation of how each status is determined based on certain
+conditions as exposed by the REST API.
+
+
+#### Running
+This is the *default* status. It is true unless one of the other conditions are
+met.
+
+-----------
+
+#### Deploying
+An application that has one ore more active deployments. True when the `v2/apps`
```

Comment:
I'm not sure I understand what you mean. The UI bases all its reasoning on the fact that the endpoint returns the deployments information. If that's not true then we can't determine the Deploying state anymore. 

How about:
```
#### Deploying
Whenever a change to the application has been requested. Marathon is performing the required actions,
which haven't completed yet. True when the `v2/apps` endpoint returns a JSON response 
with `app.deployments.length > 0`.